packages/webpack-cli/lib/utils/Compiler.js

```diff
@@ -15,17 +16,24 @@ class Compiler {
 
         compilation.hooks.beforeRun.tap('webpackProgress', () => {
             if (outputOptions.progress) {
-                let tmpMsg;
+                process.stdout.write("\n");
                 const defaultProgressPluginHandler = (percent, msg) => {
                     percent = Math.floor(percent * 100);
-                    if (percent === 100) {
-                        msg = 'Compilation completed';
+                    process.stdout.clearLine();
+                    process.stdout.cursorTo(0);
+                    if (percent !== undefined) {
+                        process.stdout.write(' (')
+                        for(let i = 0; i <= 100; i+=10) {
```

Comment:
```suggestion
                        for (let i = 0; i <= 100; i+=10) {
```

Reply:
Is prettier not running for this file?

Reply:
Yup I think so, let me check.

Reply:
Probably you did a commit and skipped the git hooks?

Reply:
I had to run eslint with `--no-ignore` for it to lint.